Classes/Controller/PasswordManagementController.php

```diff
@@ -80,21 +82,29 @@ class PasswordManagementController extends ActionController
      * @throws \Neos\Neos\Exception
      * @Flow\SkipCsrfProtection
      */
-    public function requestResetAction(string $email, string $nodeIdentifier)
+    public function requestResetAction(string $email, string $nodeIdentifier): void
     {
         $account = null;
         foreach($this->settings['authenticationProviders'] as $authenticationProviderName) {
-            $account = $this->accountRepository->findActiveByAccountIdentifierAndAuthenticationProviderName($email, $authenticationProviderName);
+            $account = $this->accountRepository->findByAccountIdentifierAndAuthenticationProviderName($email, $authenticationProviderName);
             if ($account !== null) {
                 break;
             }
+
+            $this->emitRequestedAccountForResetIsNotFound($email, $authenticationProviderName);
```

Comment:
This event is emitted too early here in the for loop, must be moved behind the for loop, else only one authProvider is checked